### DISCUSSION TITLE
In PRO plan. Failed to pause project: Project is not free-tier. Please downgrade it to free-tier first and try again.

### DISCUSSION THREAD

**Discussion by andreycattalin** [supabase/supabase#18696] [Questions] [answered]
I have 2 projects in the same organization.
Wen I try to pause one of them: "Failed to pause project: Project is not free-tier. Please downgrade it to free-tier first and try again."

**Comment by GaryAustin1** [comment, marked as answer]
You will need to transfer the project to a new organization first.
